### ISSUE TITLE
Keyboard navigation instructions don't work well in code-input elements with border-radius rounded corners

### ISSUE THREAD

**Issue by WebCoder49** [WebCoder49/code-input#180]
e.g.:

<img width="239" height="122" alt="Rounded corner nearly cuts off first letter of keyboard navigation instructions (black bar at top of element describing 'Tab and Shift-Tab currently for indentation...')" src="https://github.com/user-attachments/assets/c243289f-6016-4a90-b833-0ae4a79f08e2" />
